### PR TITLE
Truncate long source context lines

### PR DESCRIPTION
**Pull request by LewisJEllis**
This should address most of the cases causing #153.

/cc @mattrobenolt @MaxBittker

### CODE REVIEW

File: test/raven.utils.js [hunk 1]
```diff
@@ -121,6 +121,29 @@ describe('raven.utils', function () {
...
+      var callback = function (frames) {
+        var frame = frames.pop();
+
+        frame.pre_context.should.be.an.instanceOf(Array);
+        frame.pre_context[frame.pre_context.length - 1].trim().should.endWith('{snip}');
+        frame.context_line.should.be.type('string');
+        frame.context_line.trim().should.startWith('{snip}');
+        frame.context_line.trim().should.endWith('{snip}');
+        frame.post_context.should.be.an.instanceOf(Array);
+        frame.post_context[0].trim().should.endWith('{snip}');
```
Comment (benvinegar) [inline test/raven.utils.js]: I feel it's not sufficient to just test for presence of `{snip}` – can we improve this test to also assert the content of the snipped code is as expected (e.g. what appears between `{snip}`)?

**Comment by LewisJEllis** [comment]
@benvinegar updated the test. I just check the length and the presence of the `throw` statement; checking exact string equality feels like tautologically testing that the code does what it does. Let me know if there was any other check you had in mind.

**Comment by benvinegar** [comment]
It's an improvement.

> feels like tautologically testing that the code does what it does.

Yes.

**Comment by dpehrson** [comment]
Has this been released yet? Just signed up for sentry (looks awesome) and this was unfortunately the first thing I hit.

We deploy an isomorphic react app that leverages webpack asset url hashing so we webpack the server code as well.

**Comment by benvinegar** [comment]
Not yet. We'll publish something soon.

**Comment by joevbruno** [comment]
@benvinegar any progress / timeline on this?

**Comment by benvinegar** [comment]
@joevbruno – I wanted to do this earlier in the week, but now that it's Friday I'd rather wait until the weekend is over first. There's a bunch of stuff on master that could be problematic, and even this truncation change could cause some issues to group differently. We want to be on-hand just in case.

**Comment by joevbruno** [comment]
@benvinegar Understood. Hopefully it will happen sometime this week

**Comment by MaxBittker** [comment]
this has been released @joevbruno let me know if you have any problems with it!

**Pull request merged**